src/Sylius/Bundle/CoreBundle/composer.json

```diff
@@ -33,6 +33,7 @@
         "jms/serializer-bundle": "^3.5",
         "knplabs/gaufrette": "^0.8",
         "knplabs/knp-gaufrette-bundle": "^0.7",
```

Comment:
shouldn't we remove it? But we should maintain integration with gaufrette if lib is present